src/lib/adamant-api.js

```diff
@@ -13,13 +13,22 @@ Queue.configure(Promise)
 /** Promises queue to execute them sequentially */
 const queue = new Queue(1, Infinity)
 
-/** @type {{privateKey: Buffer, publicKey: Buffer}} */
-let myKeypair = { }
-let myAddress = null
-
 /** Lists cryptos for which addresses are currently being stored to the KVS */
 const pendingAddresses = { }
 
+/** Vuex selectors **/
+const user = {
```

Comment:
What does store do here? `adamant-api` is a lower-level layer and should know nothing about the store.